sdk/lib/atl/cstringt.h

```diff
@@ -797,8 +816,14 @@ class CStringT :
     }
 
 
+    BSTR AllocSysString() const
+    {
+        return StringTraits::AllocSysString(CThisSimpleString::GetString(), CThisSimpleString::GetLength());
```

Comment:
Dunno if it's the general style of ATL files, maybe going to new line to put the second "parameter" of this AllocSysString call would be nice?

Reply:
I followed the style of this file, see f.e.:
https://git.reactos.org/?p=reactos.git&a=search&h=HEAD&st=grep&sr=1&s=CThisSimpleString%3A%3ACopyCharsOverlapped